programming/opencv-installation.md

```diff
@@ -0,0 +1,33 @@
+
+# Opencv Installation Process 
+Here are the steps, that you will **run in a terminal shell** that will get you ready to run the opencv library and vision track!
+## For Linux 
+****************************** 
+Make sure your version of python is up to date 
+```
+sudo apt-get install python3-pip
+```
+Install the Opencv Library
+```
+sudo apt-get install python-opencv
+```
+Install pipenv so we can create a virtual enviroment. A virtual enviroment means the libraries we install only happen in that directory, and won't effect code in other "places" or folders of our computer
+```
+sudo pip3 install pipenv
+```
+Either manually create a folder or create a folder called vt-exp through the terminal:
+```
+mkdir vt-xp ; cd vt-xp
+```
+Cd into that project 
+```
+cd vt-exp
+```
+Create a new project using python version three
+```
+pipenv install --three
```

Comment:
Correct me if I'm wrong, but if you installed pipenv using python3/pip3, it defaults to `--three`?